ui/components/component-library/button-primary/README.mdx

```diff
@@ -0,0 +1,55 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { ButtonPrimary } from './button-primary';
+
+# ButtonPrimary
+
+The `ButtonPrimary` is an extension of `ButtonBase` to support primary styles.
+
+<Canvas>
+  <Story id="ui-components-component-library-button-primary-button-primary-stories-js--default-story" />
+</Canvas>
+
+## Props
+
+The `ButtonPrimary` accepts all props below as well as all [Box](/docs/ui-components-ui-box-box-stories-js--default-story#props) and [ButtonBase](/docs/ui-components-component-library-button-base-button-base-stories-js--default-story#props) component props
+
+<ArgsTable of={ButtonPrimary} />
+
+### Size
+
+Use the `size` prop and the `BUTTON_SIZES` object from `./ui/helpers/constants/design-system.js` to change the size of `ButtonPrimary`. Defaults to `BUTTON_SIZES.MD`
```

Comment:
Update to general design-system const. Could also add the same note from button-base

```suggestion
Use the `size` prop and the `SIZES` object from `./ui/helpers/constants/design-system.js` to change the size of `ButtonPrimary`. Defaults to `SIZES.MD`

Optional: `BUTTON_SIZES` from `./button-base` object can be used instead of `SIZES`.
```